index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Digital Marketing Summit</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="navbar">
+                <div class="container nav-container">
+                    <input class="checkbox" type="checkbox" name="" id="" />
+                    <div class="hamburger-lines">
+                        <span class="line line1"></span>
+                        <span class="line line2"></span>
+                        <span class="line line3"></span>
+                    </div>
+
```

Comment:
According tho the **project requirements**:

> You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
> 
> Colors.
> Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
> Layout: composition and space between elements.
> 

- [ ] In the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) there are two `navbars` one at the top with social media links and language settings and the second on below with `nav links` for the website. As mentioned by the previous reviewer you are missing a `navbar` on the top (with social media links and language settings). Please make sure to include this` navbar` in both pages i.e. `home` and `about` page

Your design:

<img width="1504" alt="Screenshot 2023-04-20 at 11 29 42 AM" src="https://user-images.githubusercontent.com/44798044/233286911-de9899f4-00df-42a4-b6f4-f0a919e11b46.png">


Cendy shin design:

<img width="636" alt="Screenshot 2023-04-20 at 11 29 24 AM" src="https://user-images.githubusercontent.com/44798044/233286959-86484061-5746-43ab-8129-b16654176596.png">